tests/Integration/NotifierTest.php

```diff
@@ -105,23 +106,49 @@ public function test_sendAlertsPerEmailToRecipient()
 
     public function test_sendAlertsPerEmailToRecipient_shouldUseDifferentSubjectDependingOnPeriod()
     {
-        $this->assertDateInSubject('week', 'week December 21 - 27, 2009');
-        $this->assertDateInSubject('day', 'Thursday, December 31, 2009');
-        $this->assertDateInSubject('month', 'December 2009');
+        $this->assertDateInSubject('week', $this->getMockMail('week')->getSubject());
```

Comment:
@adaqus This is an integration test, it is designed to test the subject of email is correct. In this change,  It's become some kind of unit test but does not actually test the email subject is correct. As you are now mocking the email subject, the integration test does not test anymore that email subject is correct.